src/instructions.spec.ts

```diff
@@ -469,6 +469,19 @@ describe('Cortex-M0+ Instruction Set', () => {
     expect(registers.V).toEqual(false);
   });
 
+  it('should execute an `cmp	r2, r0` instruction', async () => {
```

Comment:
Let's make the test name more specific, like: "should execute an `cmp	r2, r0` instruction and not set any flags when r0=0xb71b0000 and r2=0x00b71b00"